[inputs.diskio] Panic in gopsutil

### Relevant telegraf.conf

```toml
[[inputs.diskio]]
  devices = ["ada?", "da?"]
```

### Logs from Telegraf

```text
2025-08-23T11:07:30Z E! FATAL: [inputs.diskio] panicked: reflect: call of reflect.Value.Field on uint32 Value, Stack:
goroutine 124 [running]:
github.com/influxdata/telegraf/agent.panicRecover(0x87cc392c0)
	github.com/influxdata/telegraf/agent/agent.go:1202 +0x6d
panic({0xcaa020?, 0x87d9081c8?})
	runtime/panic.go:792 +0x132
reflect.Value.Field({0x8f0160?, 0x87cdc3608?, 0x87cdc3608?}, 0x32d2b00?)
	reflect/value.go:1272 +0xcf
github.com/shirou/gopsutil/v4/internal/common.(*decoder).value(0x87cdc3598, {0x1f5ee60?, 0x87cdc3608?, 0x120?})
	github.com/shirou/gopsutil/v4@v4.25.7/internal/common/binary.go:475 +0x8be
github.com/shirou/gopsutil/v4/internal/common.Read({0x314eae0, 0x87d96ed50}, {0x3232af0, 0x11fd91a0}, {0x680600, 0x87cdc3608})
	github.com/shirou/gopsutil/v4@v4.25.7/internal/common/binary.go:221 +0x8c5
github.com/shirou/gopsutil/v4/disk.parsedevstat({_, _, _})
	github.com/shirou/gopsutil/v4@v4.25.7/disk/disk_freebsd.go:158 +0xfc
github.com/shirou/gopsutil/v4/disk.IOCountersWithContext({0x32012c0, 0x11fd91a0}, {0x0, 0x0, 0x0?})
	github.com/shirou/gopsutil/v4@v4.25.7/disk/disk_freebsd.go:119 +0x1a7
github.com/shirou/gopsutil/v4/disk.IOCounters(...)
	github.com/shirou/gopsutil/v4@v4.25.7/disk/disk.go:84
github.com/influxdata/telegraf/plugins/common/psutil.(*SystemPS).DiskIO(0x87c557300?, {0x0?, 0x0?, 0x0?})
	github.com/influxdata/telegraf/plugins/common/psutil/ps.go:195 +0x30
github.com/influxdata/telegraf/plugins/inputs/diskio.(*DiskIO).Gather(0x87cc39200, {0x3247420, 0x87cf1c820})
	github.com/influxdata/telegraf/plugins/inputs/diskio/diskio.go:73 +0x6d
github.com/influxdata/telegraf/models.(*RunningInput).Gather(0x87cc392c0, {0x3247420, 0x87cf1c820})
	github.com/influxdata/telegraf/models/running_input.go:260 +0x251
github.com/influxdata/telegraf/agent.(*Agent).gatherOnce.func1()
	github.com/influxdata/telegraf/agent/agent.go:590 +0x58
created by github.com/influxdata/telegraf/agent.(*Agent).gatherOnce in goroutine 14
	github.com/influxdata/telegraf/agent/agent.go:588 +0xf7

goroutine 1 [sync.WaitGroup.Wait]:
sync.runtime_SemacquireWaitGroup(0x87d927680?)
	runtime/sema.go:110 +0x25
sync.(*WaitGroup).Wait(0x87d906010?)
	sync/
```

### System info

Telegraf 1.35.4, FreeBSD 14.2

### Docker

_No response_

### Steps to reproduce

1. Try to run Telegraf with the config above on FreeBSD

### Expected behavior

No panics

### Actual behavior

Telegraf panics on start

### Additional info

It looks like https://github.com/shirou/gopsutil/issues/1898 is probably related to this. Also, Telegraf 1.35.3 doesn't have this bug.

same thing for me on FreeBSD 13.  I tried with the git version a custom version of telegraf, but no luck
[global_tags]
[agent]
     interval = "10s"
     round_interval = true
     metric_batch_size = 1000
     metric_buffer_limit = 10000
     collection_jitter = "0s"
     flush_interval = "10s"
     flush_jitter = "0s"
     precision = ""
     debug = false
     quiet = false
     hostname = ""
     omit_hostname = false

#[[outputs.file]]
#     files = ["stdout"]


[[inputs.diskio]]
devices = ["da0"]
 ./telegraf --test --config /usr/local/etc/telegraf2.conf
2025-08-30T15:24:03Z I! Loading config: /usr/local/etc/telegraf2.conf
2025-08-30T15:24:03Z I! Starting Telegraf 1.36.0-a288d7da (customized) brought to you by InfluxData the makers of InfluxDB
2025-08-30T15:24:03Z I! Available plugins: 16 inputs, 0 aggregators, 0 processors, 5 parsers, 1 outputs, 0 secret-stores
2025-08-30T15:24:03Z I! Loaded inputs: diskio
2025-08-30T15:24:03Z I! Loaded aggregators:
2025-08-30T15:24:03Z I! Loaded processors:
2025-08-30T15:24:03Z I! Loaded secretstores:
2025-08-30T15:24:03Z W! Outputs are not used in testing mode!
2025-08-30T15:24:03Z I! Tags enabled: host=backup4.sentex.ca
2025-08-30T15:24:03Z W! [agent] The default value of 'skip_processors_after_aggregators' will change to 'true' with Telegraf v1.40.0! If you need the current default behavior, please explicitly set the option to 'false'!
panic: reflect: call of reflect.Value.Field on uint32 Value

goroutine 69 [running]:
reflect.Value.Field({0xdf1940?, 0x86c17b628?, 0x86c17b628?}, 0x11b0668?)
        /root/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.25.0.freebsd-amd64/src/reflect/value.go:1261 +0xcf
github.com/shirou/gopsutil/v4/internal/common.(*decoder).value(0x86c17b5b8, {0xf7afa0?, 0x86c17b628?, 0x120?})
        /root/go/pkg/mod/github.com/shirou/gopsutil/v4@v4.25.7/internal/common/binary.go:475 +0x8cb
github.com/shirou/gopsutil/v4/internal/common.Read({0x118c720, 0x86c1cb500}, {0x119d290, 0x1953a40}, {0xdd2740, 0x86c17b628})
        /root/go/pkg/mod/github.com/shirou/gopsutil/v4@v4.25.7/internal/common/binary.go:221 +0x605
github.com/shirou/gopsutil/v4/disk.parsedevstat({_, _, _})
        /root/go/pkg/mod/github.com/shirou/gopsutil/v4@v4.25.7/disk/disk_freebsd.go:158 +0xfc
github.com/shirou/gopsutil/v4/disk.IOCountersWithContext({0x1196630, 0x1953a40}, {0x86c618390, 0x1, 0x0?})
        /root/go/pkg/mod/github.com/shirou/gopsutil/v4@v4.25.7/disk/disk_freebsd.go:119 +0x1a5
github.com/shirou/gopsutil/v4/disk.IOCounters(...)
        /root/go/pkg/mod/github.com/shirou/gopsutil/v4@v4.25.7/disk/disk.go:84
github.com/influxdata/telegraf/plugins/common/psutil.(*SystemPS).DiskIO(0x0?, {0x86c618390?, 0x0?, 0x0?})
        /home/mdtancsa/telegraf/plugins/common/psutil/ps.go:195 +0x30
github.com/influxdata/telegraf/plugins/inputs/diskio.(*DiskIO).Gather(0x86c328000, {0x119e500, 0x86c0d2380})
        /home/mdtancsa/telegraf/plugins/inputs/diskio/diskio.go:73 +0x6d
github.com/influxdata/telegraf/agent.(*Agent).testRunInputs.func2(0x86c3280c0)
        /home/mdtancsa/telegraf/agent/agent.go:531 +0x2ca
created by github.com/influxdata/telegraf/agent.(*Agent).testRunInputs in goroutine 67
        /home/mdtancsa/telegraf/agent/agent.go:500 +0xd1

--- a/go.mod
+++ b/go.mod
@@ -188,7 +188,7 @@ require (
 	github.com/santhosh-tekuri/jsonschema/v5 v5.3.1
 	github.com/seancfoley/ipaddress-go v1.7.1
 	github.com/sensu/sensu-go/api/core/v2 v2.16.0
-	github.com/shirou/gopsutil/v4 v4.25.7
+	github.com/shirou/gopsutil/v4 v4.25.8
 	github.com/showwin/speedtest-go v1.7.10
 	github.com/signalfx/golib/v3 v3.3.54
 	github.com/sijms/go-ora/v2 v2.9.0

